common/scripts/metallb.yaml

```diff
@@ -307,45 +358,63 @@ spec:
               valueFrom:
                 fieldRef:
                   fieldPath: status.podIP
+            # needed when another software is also using memberlist / port 7946
+            # when changing this default you also need to update the container ports definition
+            # and the PodSecurityPolicy hostPorts definition
+            #- name: METALLB_ML_BIND_PORT
+            #  value: "7946"
             - name: METALLB_ML_LABELS
               value: "app=metallb,component=speaker"
-            - name: METALLB_ML_NAMESPACE
-              valueFrom:
-                fieldRef:
-                  fieldPath: metadata.namespace
             - name: METALLB_ML_SECRET_KEY
               valueFrom:
                 secretKeyRef:
                   name: memberlist
                   key: secretkey
-          image: metallb/speaker:v0.9.3
-          imagePullPolicy: Always
+          image: quay.io/metallb/speaker:v0.12.1
```

Comment:
Need to avoid `quay.io`, they are unreliable. I've mirror to gcr.io/istio-testing/metallb/speaker